common/common.go

```diff
@@ -154,10 +155,32 @@ func (l *PrivateNetList) String() string {
 
 func (l *PrivateNetList) Set(value string) error {
 	if l.mapping == nil {
-		l.mapping = make(map[string]bool)
+		l.mapping = make(map[string]string)
 	}
 	for _, s := range strings.Split(value, ",") {
-		l.mapping[s] = true
+		netArgsPair := strings.Split(s, ":")
+		netName := netArgsPair[0]
+
+		if netName == "" {
+			return fmt.Errorf("netname must not be empty")
+		}
+		if _, duplicate := l.mapping[netName]; duplicate {
+			return fmt.Errorf("found duplicate netname %q", netName)
+		}
+
+		switch {
+		case len(netArgsPair) == 1:
+			l.mapping[netName] = ""
+		case len(netArgsPair) == 2:
+			if netName == "all" || strings.HasPrefix(netName, "default") {
+				return fmt.Errorf("arguments are not supported by special netname %q", netName)
```

Comment:
I think you should be able to pass args to "default". It's just a net predefined by rkt or even user, if placed into /etc/rkt/net.d